packages/manager/modules/sms/src/sms/batches/details/routing.js

```diff
@@ -0,0 +1,51 @@
+import moment from 'moment';
+
+import component from './telecom-sms-batches-details.component';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('sms.service.batches.details', {
+    url: '/:batchId',
+    views: {
+      'smsView@sms.service': {
+        component: component.name,
+      },
+    },
+    params: {
+      previousState: '^',
+    },
+    resolve: {
+      batch: /* @ngInject */ ($transition$, batches) =>
+        batches.find(({ id }) => id === $transition$.params().batchId),
```

Comment:
batchId could be in its own resolve

Reply:
ok but I don't use it by itself

Reply:
Ok but it would ease reuse in controller and other resolve if needed